### PR TITLE
React/Typescript setup

### PR DESCRIPTION
**Pull request by lvreynoso**
### Description

Sets up the React/Typescript frontend and configures Webpack to place the built files into the Flask app's `static/` folder. Modifies Flask to use the `static/index.html` entrypoint. Configures Flask to not cache files in a dev environment (needed to play nice with Webpack's auto-building of updated files).

Screenshot from 2021-01-13 17-25-43

#### Issue [ch64849]
[ch64849](https://app.clubhouse.io/genepi/stories/space/64849)

### Test plan

Make sure these environment variables are set on your machine somewhere:

```
export FLASK_APP="/path/to/git/dir/covidr/src/py/covidr"
export FLASK_ENV="development"
```

Start the Webpack watcher with `cd src/ts && npm start`.
Separately, start flask with `cd src/py && pip install -e . && flask run`.

Open up your browser and go to `localhost:5000`. Change stuff in the React app, reload the page, and watch the magic happen!

### CODE REVIEW

File: src/ts/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (ttung) [inline src/ts/.gitignore]: this has duplicates with the top-level .gitgnore.

File: src/ts/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
```
Comment (ttung) [inline src/ts/README.md:7]: Just to clarify, do we run this in `covidr/` or `covidr/src/ts/`?

**Review by ttung — COMMENTED** [review]
This looks great!

File: src/ts/README.md (hunk 2 above)
Comment (lvreynoso) [reply src/ts/README.md:7]: `covidr/src/ts/`

File: src/ts/.gitignore (hunk 1 above)
Comment (lvreynoso) [reply src/ts/.gitignore]: removed this file, don't know how it snuck in

File: src/py/covidr/config/config.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    DATABASE_URI = ""
```
Comment (ttung) [inline src/py/covidr/config/config.py:4]: We'll have to reconcile this and https://github.com/chanzuckerberg/covidr/pull/8/files#diff-e21eb535636415b0f95d7d87b7e9f5164945ec1ff0f00116d547b2028273b261.  Proposal: let's land both as is, and we can refactor.

**Review by ttung — APPROVED** [review]
This looks good to me.  Pls see comments.

Can be in a subsequent PR, but can you:

1.  add a github actions rule to run eslint?
2. add a github actions to test that npm run build passes?
3. add something to README.md for a ts/js newbie like myself to get started with this?

**Pull request merged**